Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN go build -o /bin/deplist ./cmd/deplist/
 
 FROM registry.fedoraproject.org/fedora:34
 RUN dnf install -y \
-    golang-bin-1.16 \
+    golang-bin \
```

Comment:
The default golang-bin version in fedora 34 is 1.16. I don't think this changes much aside from making it less obvious which go version is installed, based on the Dockerfile.